### PR TITLE
Apply component template to commodore itself

### PR DESCRIPTION
**Pull request by corvus-ch**
## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Update the ./CHANGELOG.md.

### CODE REVIEW

**Comment by corvus-ch** [comment]
Is this the moment to drop the DCO?

**Comment by tobru** [comment]
Yes, I'd drop DCO.

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 * `compile` and `clean` commands were moved to the `catalog` command group ([#122])
 * `new-component` command was moved to the `component` command group ([#122])
 * Jsonnet library file extension to `.libsonnet` in component template ([#137])
+* Applied the component template to Commodore itself [#145]
```
Comment (srueg) [inline CHANGELOG.md:25]:
```suggestion
* Applied the component template to Commodore itself ([#145])
```

**Comment by corvus-ch** [comment]
DCO dropped.

**Pull request merged**